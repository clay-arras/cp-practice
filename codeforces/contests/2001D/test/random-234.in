1
18
14 24 19 24 11 7 8 4 18 16 16 9 15 22 7 8 17 22
