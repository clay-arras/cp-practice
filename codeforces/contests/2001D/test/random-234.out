13
14 19 24 11 8 4 18 16 9 15 22 7 17
